README.md

```diff
@@ -1,138 +1,119 @@
-  * Multifactor authentication with OTP, U2F
-  * Federation with OpenID Connect 1.0 Protocol (RP), OAuth 2.0 Protocol (RP)
-  * Identity Brokering
-* Identity & Access Management
-  * B2C, B2B, B2E, M2M identities
-  * Authorization via Role Based Access Control (RBAC)
-  * Delegation of roles to other organizations for self-management
-  * Management roles
-  * User self-service workflows
-  * User register workflow
-* Strong audit trail for all IAM resources
-* Privatelabeling
-  * Custom branding
-  * Custom texts
-  * Hosted login
-  * Personal profile
-* Integration
-  * API-first
-  * Actions for custom code execution
+```bash
+COMPOSE_DOCKER_CLI_BUILD=1 DOCKER_BUILDKIT=1 docker compose -f ./build/local/docker-compose.yml up
+```
 
-## Showcase
+### Run on your machine
 
-### Passwordless Login
+<!--  TODO: download/install zitadel  -->
 
-Use our login widget to allow easy and sucure access to your applications and enjoy all the benefits of passwordless (FIDO 2 / WebAuthN):
+<!--  TODO: add basic start-with-init command -->
+<!--  TODO: insecure, should we write logs to a file? -->
+<!-- TODO: what will be printed? is it possible to open the browser and show the login screen? -->
 
-* works on all modern platforms, devices, and browsers
-* phishing resistant alternative
-* requires only one gesture by the user
-* easy [enrollment](https://docs.zitadel.ch/docs/manuals/user-factors) of the device during registration
+First of all you need to start the cockroach database [using their guide to start a cluster](https://www.cockroachlabs.com/docs/v21.2/start-a-local-cluster)
 
-![passwordless-windows-hello](https://user-images.githubusercontent.com/1366906/118765435-5d419780-b87b-11eb-95bf-55140119c0d8.gif)
+After the database is up and running you can start ZITADEL:
 
-### Admin Console
+```bash
+zitadel start-with-init
+```
 
-Use [Console](https://docs.zitadel.ch/docs/manuals/introduction) or our [APIs](https://docs.zitadel.ch/docs/apis/introduction) to setup organizations, projects and applications.
+### Client libraries
 
-Register new applications
-![OIDC-Client-Register](https://user-images.githubusercontent.com/1366906/118765446-62064b80-b87b-11eb-8b24-4f4c365b8c58.gif)
+| Language | Client | ManagmentAPI | AdminAPI | AuthAPI | Thanks to the maintainers |
+|----------|--------|--------------|----------|---------|---------------------------|
+| .NET       | [zitadel-net](https://github.com/caos/zitadel-net) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
+| Dart       | [zitadel-dart](https://github.com/caos/zitadel-dart) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
+| Elixir     | [zitadel_api](https://github.com/jshmrtn/zitadel_api) | ✔️ | ✔️ | ✔️ | [jshmrtn 🙏🏻](https://github.com/jshmrtn) |
+| Go         | [zitadel-go](https://github.com/caos/zitadel-go) | ✔️ | ✔️ | ✔️ | ZITADEL |
+| Rust       | [zitadel-rust](https://crates.io/crates/zitadel) | ✔️ | ✔️ | ✔️ | [buehler 👑](https://github.com/buehler) |
```

Comment:
Why not show important but missing languages? IMO would be useful to make others aware that this is something they can contribute in a meaningful way.

Reply:
added js, python, jvm